packages/commonwealth/client/scripts/views/components/FeedDiscovery/FeedDiscovery.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import {Helmet} from "react-helmet";
+
+export default function FeedDiscovery() {
```

Comment:
we do not use function syntax for components, so please use 

`const FeedDiscovery = () => {}`

Here is a doc with set of rules we should follow creating new components. 

https://github.com/hicommonwealth/commonwealth/blob/master/knowledge_base/React-Best-Practices-And-Improvements.md

sidenote: Not 100% of the codebase follow that rules, as we still have "legacy" code

Reply:
To add on to this, could we add this as an eslint rule to enforce it on the CI?

https://www.npmjs.com/package/eslint-plugin-react-prefer-function-component

Reply:
You have done that before, right? My IDE is showing a warning with this rule in this file

Reply:
Good catch Marcin. Sorry for the confusion everyone. It seems we weren't linting our tsx files due to a change back in Janurary. I have put up a PR to fix it:
#7086

Reply:
> You have done that before, right? My IDE is showing a warning with this rule in this file

Interesting my IDE isn't showing our eslint errors which is annoying but it also didn't fail CI.  Maybe it's a warning? I can track this down if it's a major issue.

Reply:
> Interesting my IDE isn't showing our eslint errors which is annoying but it also didn't fail CI. Maybe it's a warning? I can track this down if it's a major issue.

Yes, this was an issue with the CI pipeline. I have fixed it, and it has been merged in, so it should fail the CI now.